src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -202,19 +231,21 @@ def create_api(self):
         self.log.info("Successfully created API")
         return api_id
 
-    def create_resource(self, parent_id=""):
+    def create_or_update_resource(self, parent_id=""):
```

Comment:
There is at least one call to the method `create_resource()` that needs to be updated to the new method name.